semgrep_output_v0.atd

```diff
@@ -119,15 +119,31 @@ type core_match_extra
 type dataflow_trace
   <ocaml attr="deriving show">
   <python decorator="dataclass(frozen=True)"> = {
-  ?taint_source: location list option;
+  ?taint_source: taint_source option;
   (* Intermediate variables which are involved in the dataflow. For taint, this
    * explains how the taint flows from the source to the sink. *)
-  ?intermediate_vars: location list option;
+  ?intermediate_vars: intermediate_var list option;
   (* For now, omitting the taint sink, since it's redundant data except for
    * certain cases that only appear with DeepSemgrep. We can add it in later
    * without breaking consumers. *)
 }
 
+type taint_source
+  <ocaml attr="deriving show">
+  <python decorator="dataclass(frozen=True)"> = {
+  location: location;
+  abstract_content: string;
```

Comment:
Why do we need those abstract_content?
Those abstract_content were hacks used a long time ago when the rule engine was split between OCaml and Python.
I'm not even sure we need them anymore in metavar_value.

Reply:
@ajbt200128 specifically requested it to make the taint trace display easier in the App